README.md

```diff
@@ -1,5 +1,60 @@
+
 # Money tracker tech test
 
+[![Build Status](https://travis-ci.org/Corina/money_tracker_tech_test.svg?branch=master)](https://travis-ci.org/Corina/money_tracker_tech_test)
+[![Coverage Status](https://coveralls.io/repos/github/Corina/money_tracker_tech_test/badge.svg?branch=master)](https://coveralls.io/github/Corina/money_tracker_tech_test?branch=master)
+
+## Synopsis
+
+Money tracker is a Node.js app that helps users keep track of money spent and earned.
+
+## User stories
+
+**As a user**,
+
+So I can manage my money,  
+I want to open an account
+
+So I can know what I spent,   
+I want to be able to record money spent from my account
+
+So I can know what I earned,  
+I want to be able to record money deposited on my account
+
+So I know what transactions took place on my account,  
+I want to be able to see a list of transactions
+
+So I know when each transaction took place,  
+I want to see the list of transactions and the date when each took place
+
+## Built
+
+Built using Node.js.
+
+## Testing
+
+Tested using Jasmine. Added a custom_reporter.js file to customize testing output, making text from `describe` and `it` blocks visible and adding indentation.
+
+![id](https://raw.githubusercontent.com/Corina/money_tracker_tech_test/master/assets/images/Screen%20Shot%202017-08-07%20at%2011.24.58%20PM.png)
+
+
+## Installation
+Clone this repo. Run examples from `usageExamples.js` by typing `node usageExamples.js`. Type `node` to access the REPL environment and play with the program.
+To run tests type `jasmine` in the command line.
+
+![id](https://raw.githubusercontent.com/Corina/money_tracker_tech_test/master/assets/images/Screen%20Shot%202017-08-08%20at%201.10.18%20PM.png)
+
+## Approach
```

Comment:
If you write about your approach in a real tech test, focus on the architectural details.  Some of your description here is doing that, which is great.  (eg the stuff about the format of the date.) But it would be great to hear more about how you decided to separate the features amongst the classes.  How you designed the class interfaces.  That is, all the details of your OO and TDD thinking.